lbjava/README.md

```diff
@@ -18,8 +18,41 @@ Visit each link for its content
  7. [A working example: regression](doc/REGRESSION.md)
  8. [Learning Algorithms](doc/ALGORITHMS.md)
  
-**Note**
+## Using it as a dependency  
+To include LBJava in your Maven project, add the following snippet with the
+   `#version` entry replaced with the version listed in this project's pom.xml file. 
+   Note that you also add to need the
+   `<repository>` element for the CogComp maven repository in the `<repositories>` element.
+    
+```xml 
+    <dependencies>
+         ...
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>LBJava</artifactId>
+            <version>#version#</version>
+        </dependency>
+        ...
+    </dependencies>
+    ...
+    <repositories>
+        <repository>
+            <id>CogcompSoftware</id>
+            <name>CogcompSoftware</name>
+            <url>http://cogcomp.cs.illinois.edu/m2repo/</url>
+        </repository>
+    </repositories>
+```
+
+## Compiling the code 
+To compile and package the LBJava code simply run:
+
+    mvn install
+
+NB: If you need to run `mvn clean` at any point, make sure to create `target/classes` directory before 
+running `mvn compile/install` since it is required for the java-source compilation process.
 
+## Using ILP inference 
```

Comment:
Yeah I think I will update these later, when we are good place for the inference algorithms....